docs/framework/migration-guide/retargeting/throttle_concurrent_requests_per_session.md

```diff
@@ -0,0 +1,9 @@
+### Throttle concurrent requests per session
+
+|   |   |
+|---|---|
+|Details|In the .NET Framework 4.6.2 and earlier, Asp.net executes requests with the same Sessionid sequentially, and asp.net always issues the Sessionid through cookie by default. If a page takes long time to response, it will significantly degrade the server performance just by pressing F5 on the browser. In the fix, we added a counter to track the queued requests and terminates the requests when they exceed a specified limit. The default value is 50. If the the limit is reached, a warning will be logged in event log and HTTP 500 response may be recorded in IIS log.|
```

Comment:
long time --> a long time
the server performance --> server performance
terminates the requests --> terminate the requests
logged in event log and HTTP 500 --> logged in the event log, and an HTTP 500
in IIS log --> in the IIS log